code/online_challenges/src/project_euler/problem_007/problem_007.py

```diff
@@ -0,0 +1,15 @@
+def main():
+	n = 10001
+	x = 2
+	list_of_primes = []
+	
+	while (len(list_of_primes) < n):
+		if all (x % prime for prime in list_of_primes):
+			list_of_primes.append(x)
+		x += 1
+
+	print (list_of_primes[-1])
+
+
```

Comment:
Remove this extra newline

Reply:
I don't think this should be removed. According to PEP 8 style guide, we need to follow for python code in this repository, there should be 2 blank lines after class or function definition.

Reply:
@parulagg27 Looks like you're right (You also need this at the top of the file). However, I haven't really seen other files in Python follow this convention, so I would just remove the line.

Reply:
Oh, alright.